src/dialog.tsx

```diff
@@ -513,22 +554,27 @@ export default class DialogRoot extends React.Component<DialogRootProps, DialogR
 }
 
 const showDialog =
-    (showCancel = false) =>
+    (showCancel = false, forceNonNative: boolean) =>
```

Comment:
I prefer an object:
```js
const showDialog = 
    ({showCancel = false, forceNonNative = false}: {showCancel: boolean, forceNonNative: boolean}) => ...
```

This way instead of:
```js
showDialog(true, false)
```
you would read:
```js
showDialog({showCancel: true, forceNonNative: false})
```